### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by nweiyue**
Update Ui component
Add switch tabs implementation

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -156,6 +159,45 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+
+Given below is an example usage scenario and how the switch mechanism behaves at each step.
+
+Step 1. The user launches the application for the first time. The default students tab is displayed.
+
+Step 2. The user executes `switch sessions` command to view the sessions tab. `MainWindow#executeCommand()` is called and `LogicManager` calls `AtasParser#parseCommand()`. This results in the creation of a `SwitchCommandParser` to parse the user input.
+
+Step 3. `SwitchCommandParser#parse()` checks if there is an argument being passed by the user. If an argument is passed, a `SwitchCommand` will be created and `SwitchCommand#execute()` will be called by the `LogicManager`.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** If no arguments is found, a `ParseException` will be thrown and the execution terminates.
```
Comment (marcustw) [inline docs/DeveloperGuide.md]: Perhaps "If no arguments are found" ?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -156,6 +159,45 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+
+</div><br>
+
+Step 4. `SwitchCommand#execute()` will check if the argument pass is an existing tab name. If the argument is valid, a `CommandResult` containing a variable `switchTab` with the value of the target `Tab` will be returned to `MainWindow`.
```
Comment (marcustw) [inline docs/DeveloperGuide.md]: Perhaps "if the argument passed in is an"?

**Review by marcustw — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (nweiyue) [reply docs/DeveloperGuide.md]: Thanks for spotting the mistakes!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -156,6 +159,45 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+
+Step 2. The user executes `switch sessions` command to view the sessions tab. `MainWindow#executeCommand()` is called and `LogicManager` calls `AtasParser#parseCommand()`. This results in the creation of a `SwitchCommandParser` to parse the user input.
+
+Step 3. `SwitchCommandParser#parse()` checks if there is an argument being passed by the user. If an argument is passed, a `SwitchCommand` will be created and `SwitchCommand#execute()` will be called by the `LogicManager`.
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** If no argument is found, a `ParseException` will be thrown and the execution terminates.
+
+</div><br>
+
+Step 4. `SwitchCommand#execute()` will check if the argument passed in is an existing tab name. If the argument is valid, a `CommandResult` containing a variable `switchTab` with the value of the target `Tab` will be return to `MainWindow`.
```
Comment (CodingCookieRookie) [inline docs/DeveloperGuide.md:174]: Perhaps can combine "If the argument is valid" with no argument ==> If argument is invalid, a `ParseException`...?

Comment (nweiyue) [reply docs/DeveloperGuide.md:174]: Thanks for the suggestion but if the argument is invalid, it throws a CommandException but if argument is empty it throws a ParseException. Hence, this is why I separated them.

Comment (CodingCookieRookie) [reply docs/DeveloperGuide.md:174]: > Thanks for the suggestion but if the argument is invalid, it throws a CommandException but if argument is empty it throws a ParseException. Hence, this is why I separated them.

Noted thanks for clarification

**Pull request merged**